src/openforms/forms/api/validators.py

```diff
@@ -190,6 +190,19 @@ def __call__(self, attrs: dict, serializer: serializers.Serializer):
             )
 
 
+class FormStepIsApplicableIfFirstValidator:
+    def __call__(self, attrs: dict):
+        if not attrs.get("is_applicable", True) and attrs.get("order") == 0:
```

Comment:
We have to negate one of the condition anyhow. With your suggestion, if `is_applicable` is `True` and `order` is `0`, it will raise a validation error even though it is still correct

> Since `is_applicable` has a default

It has one on the model but I think it is not present in the data dict if not changed by the user

Reply:
I am an idiot, I don't know what I was thinking... nvm :P